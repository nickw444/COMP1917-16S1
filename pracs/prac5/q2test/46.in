46
5 3 9 10 -8 -9 -9 -2 -10 6 -2 -9 5 1 9 0 -8 0 -10 7 1 -5 -5 -10 -8 -4 8 -9 -8 -7 -9 -9 -3 3 -8 7 -5 -10 2 4 4 7 -3 2 -2 -5
