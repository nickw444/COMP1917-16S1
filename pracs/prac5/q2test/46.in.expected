average of array: -1.826087
